mantidproject/mantid
# Get property default values in DrILL interface

**Description of work.**

Instead of repeating the properties default values in the DrILL interface, they are now collected from the selected algorithm.

**To test:**

Open the DrILL interface, select an instrument and an acquisition mode and open the settings (button with the gear icon).
Observe that the settings are populated with their default value (see algorithm documentation).


Part of #26906 .

*This does not require release notes*: internal changes only.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
